src/server/system_services/system_store.js

```diff
@@ -740,17 +744,27 @@ class SystemStore extends EventEmitter {
             });
         });
 
-        const bulk_results = await Promise.all(Object.values(bulk_per_collection).map(
-            bulk => bulk.length && bulk.execute({ j: true })
-        ));
 
-        for (const res of bulk_results) {
-            if (res && !res.ok) {
-                dbg.error('got error on bulk execute', res.err);
-                throw new Error(res.err);
+        if (config.DB_TYPE === 'postgres') {
```

Comment:
Maybe worth hiding the different implementations in the db clients. I suggest adding a function to `DBClient` interface in [nb.d.ts](https://github.com/noobaa/noobaa-core/blob/aeb5e82334ba1526cec2016ad698045e44ba02dc/src/sdk/nb.d.ts#L649), and then you can have a different implementation for postgres\mongo. Maybe something like `execute_multiple_bulks()`.
This function should return a common `bulk_results` array that should be inspected the same for both DBs

Reply:
Done.  Good Idea.